src/mrpro/utils/_rgetattr.py

```diff
@@ -15,6 +15,6 @@
 import functools
 
 
-def rgetattr(obj, attr, *args):
+def rgetattr(obj, attr, *args):  # noqa: ANN001
     """Recursive getattr for nested attributes."""
-    return functools.reduce(lambda obj, attr: getattr(obj, attr, *args), [obj] + attr.split('.'))
+    return functools.reduce(lambda obj, attr: getattr(obj, attr, *args), [obj] + attr.split('.'))  # noqa: RUF005
```

Comment:
Didn't have the time to write a comment, when I pushed the commit, but I didn't change anything for _rgetattr because we don't use it anyway, do we? I wanted to discuss if we wanna remove it....

Reply:
I will change it to be consistent with the rest of our code. For an explanation why we should prefer the unpacking op for concatination, check the Ruff doc for rule [RUF005](https://docs.astral.sh/ruff/rules/collection-literal-concatenation/).

Still, if we don't use the function anymore, we should probably remove it...

Reply:
TIL.